proxy.php

```diff
@@ -301,34 +301,73 @@ protected function apply_filters() {
 				],
 			];
 		}
+
+		/**
+		 * If there's no aggregations in the template or if the relation isn't 'and', we are done.
```

Comment:
@felipeelia this comment makes me feel a little bit confused. Wouldn't be better `if the relation is 'or'...`?